protobuf.tmpl

```diff
@@ -1,9 +1,13 @@
 ---
-id: api
+id: protobuf-api
```

Comment:
I've done very little with the protobuf generation, so I'll differ to @sachaw to make sure these changes are working correctly.

Reply:
Yeah, that's fine, just make sure the placeholder page is updated accordingly